### PR TITLE
CNV20023: MigrionPolicy details page

### PR DESCRIPTION
**Pull request by avivtur**
Signed-off-by: Aviv Turgeman <aturgema@redhat.com>
## 📝 Description

Add details page and route for MigrationPolicy 
NOTE: Configurations Edit modal (Edit MigrationPolicy modal) will be in a follow-up PR

[UX design](https://docs.google.com/document/d/1vzFcNXIM3KZwgWbgESx-BenhDZRqS7gj1lGsQ26JqfY/edit#heading=h.l1ob50ujvmgw)
[Sub-task](https://issues.redhat.com/browse/CNV-20023)
[Issue](https://issues.redhat.com/browse/CNV-18219)

## 🎥 Demo

https://user-images.githubusercontent.com/67270715/183284834-42c9dd42-aef5-49be-93cc-ca6a16613a66.mp4

### CODE REVIEW

File: src/views/migrationpolicies/details/components/MigrationPolicyBreadcrumb/MigrationPolicyBreadcrumb.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { migrationPoliciesPageBaseURL } from 'src/views/migrationpolicies/list/utils/constants';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+
+import './MigrationPolicyBreadcrumb.scss';
+
+export const MigrationPolicyBreadcrumb: React.FC = React.memo(() => {
```
Comment (metalice) [inline src/views/migrationpolicies/details/components/MigrationPolicyBreadcrumb/MigrationPolicyBreadcrumb.tsx:10]: do we really need memo here?

File: src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/MigrationPolicyDetailsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      </a>
+      <Title headingLevel="h2" className="kv-details-section-heading">
+        {t('MigrationPolicy Details')}
+      </Title>
+      <Grid hasGutter>
+        <GridItem span={5}>
+          <DescriptionList>
+            <MigrationPolicyDescriptionItem title={t('Name')} description={mp?.metadata?.name} />
+            <>
+              <DescriptionListTermHelpTextButton /*onClick={Open editconfigmodal}*/>
```
Comment (metalice) [inline src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/MigrationPolicyDetailsSection.tsx:53]: ?

File: src/views/migrationpolicies/utils/utils.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  value ? t('YES') : t('NO');
+
+export const getBandwidthPerMigrationText = (
+  bandwidth: string | number,
+  t: TFunction,
+): string | number => {
+  if (!bandwidth || bandwidth === '0') return t('Unlimited');
+  if (typeof bandwidth === 'string') return readableSizeUnit(bandwidth);
+  return bandwidth;
+};
```
Comment (metalice) [inline src/views/migrationpolicies/utils/utils.ts:16]: can you please try to see if we can import "t" from i18n instead of passing it as an arg?

Comment (avivtur) [reply src/views/migrationpolicies/utils/utils.ts]: done, thanks for the tip :)

File: src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/MigrationPolicyDetailsSection.tsx (hunk 2 above)
Comment (avivtur) [reply src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/MigrationPolicyDetailsSection.tsx:53]: will be removed in follow-up PR :)

File: src/views/migrationpolicies/details/components/MigrationPolicyBreadcrumb/MigrationPolicyBreadcrumb.tsx (hunk 1 above)
Comment (avivtur) [reply src/views/migrationpolicies/details/components/MigrationPolicyBreadcrumb/MigrationPolicyBreadcrumb.tsx]: nope, removed thanks :)

**Pull request merged**